src/plugins/mer/meremulatormodedialog.cpp

```diff
@@ -222,7 +225,8 @@ bool MerEmulatorModeDialog::execDialog()
         m_ui->restartEmulatorCheckBox->setEnabled(false);
     }
 
-    if (m_dialog->exec() != QDialog::Accepted) {
+    int dialogResult = m_dialog->exec();
+    if (dialogResult != QDialog::Accepted) {
```

Comment:
You can fix the style while touching this :)

Reply:
Sure